compiler/rustc_parse_format/src/lib.rs

```diff
@@ -450,27 +452,30 @@ impl<'a> Parser<'a> {
 
     /// Forces consumption of the specified character. If the character is not
     /// found, an error is emitted.
-    fn must_consume(&mut self, c: char) -> Option<usize> {
+    fn consume_closing_brace(&mut self, arg: &Argument<'_>) -> Option<usize> {
         self.ws();
 
         if let Some(&(pos, maybe)) = self.cur.peek() {
-            if c == maybe {
+            if maybe == '}' {
                 self.cur.next();
                 Some(pos)
             } else {
                 let pos = self.to_span_index(pos);
                 let description = format!("expected `'}}'`, found `{maybe:?}`");
-                let label = "expected `}`".to_owned();
-                let (note, secondary_label) = if c == '}' {
+                let label = "expected `'}'`".to_owned();
```

Comment:
Having both backticks and single quotes seems weird to me. Is this the normal error message style? I feel like it'd be better to have one or the other, but this looks like a conscious change so there's probably a reason to do it this way.

Reply:
This is the only label in this error reporting path that doesn't use the double backticks, so it's just to make it more normal.

Reply:
I don't particularly like them either, but the fallout of changing them all to just single backticks is a lot more than changing this to match the rest..

Reply:
Sounds good. Consistency seems like a good reason for the change!